Triangulation/examples/Triangulation/delaunay_triangulation.cpp

```diff
@@ -1,4 +1,4 @@
-#if defined(__GNUC__) && defined(__GNUC_MINOR__) && (__GNUC__ <= 4) && (__GNUC_MINOR__ < 4)
+#if defined(__GNUC__) && not defined (__clang__) && defined(__GNUC_MINOR__) && (__GNUC__ <= 4) && (__GNUC_MINOR__ < 4)
```

Comment:
The use of `not` is not portable. It is in the standard, as [Alternative operator representations](https://en.cppreference.com/w/cpp/language/operator_alternative), but not implemented everywhere. Please replace by `!`.